content/en/armory-enterprise/quickspin/install-quick-spin.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Install Quick Spin 
+linkTitle: "Install Quick Spin"
+no_list: true
+weight: 4
+description: >
+  Use the Armory Quick Spin container to quickly get an evaluation or testing instance of Spinnaker up and running.
+exclude_search: true
+---
+
+## {{% heading "prereq" %}}
+
+
+To run the Quick `quick-spin` container you need a hosting machine that can support a running instance with 
+  - 4 vCPU
+  - 8GB Memory
+  - 2GB Disk space
+
+The hosting machine must have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed. Docker Compose is installed with Docker Desktop.
+
+## Usage
+
+To run a local instance of `quick-spin` using [docker compose](https://docs.docker.com/compose/):
+
+1. Clone the [Quick Spin repository](https://github.com/armory-io/quick-spin).
+   
+```shell
+git clone https://github.com/armory-io/quick-spin.git
+```
+2. From the terminal navigate to the root of the cloned repository.
+   
+```shell
+cd quick-spin
+```
+3. Run the following command:
```

Comment:
Some of your bullet points end in '.' while others end in ':' - suggest choosing one or the other.